ApiTests/Tests/UI/BaseTest.cs

```diff
@@ -1,10 +1,17 @@
-﻿using TestProject.Utils;
+﻿using OpenQA.Selenium.Support.UI;
+using TestProject.Utils;
 
 namespace ApiTests.Tests.UI
 {
     internal class BaseTest
     {
         protected static readonly string partialTextXpath = "//*[contains(text(),'{0}')]";
+        private static readonly string randomValue = Guid.NewGuid().ToString();
+
+        public static string GetRandomValue()
+        {
+            return randomValue;
+        }
```

Comment:
Should be a part of a separate class, such as RandomUtils 
This one is a part of SOLID in this case - Single Responsibility. If BaseTest's logic is all about driver setup (incl. teardown) then definitely this method is of no use in here, but if we move it to a separate class we can reuse it easily (for API tests for example)